submissions/AQuaZar/port-sniffer/sniffer.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import print_function  # Compatibility with python2
+import sys
+import argparse
+import socket
+
+
+def range_type(astr, min=0, max=65535):
+    value = int(astr)
+    if min <= value <= max:
```

Comment:
I wouldn't use `min` and `max` as variable names because you overwrite `min` and `max` functions